core.py

```diff
@@ -374,6 +374,7 @@ def run_preprocess_script(
     config = get_config()
     per = 3.0 if config.is_half else 3.7
     preprocess_script_path = os.path.join("rvc", "train", "preprocess", "preprocess.py")
+    os.makedirs(os.path.join(logs_path, model_name), exist_ok=True)
```

Comment:
This line is not necessary

Reply:
Yeah, I forgot to remove it, thanks for reminding me